src/olympia/search/filters.py

```diff
@@ -441,13 +441,13 @@ def convert_to_hsl(self, hexvalue):
             hexvalue = ''.join(2 * c for c in hexvalue)
         try:
             rgb = tuple(bytearray.fromhex(hexvalue))
-        except ValueError:
-            rgb = (0, 0, 0)
+        except ValueError as err:
+            raise ValueError(gettext('Expected a hex string color code.')) from err
```

Comment:
The other filters in this file simply use `gettext('Invalid "%s" parameter.' % self.query_param)`, let's use that to be consistent.

(It doesn't specify the error but does specify the param name, which should be enough given that it's should be raised to users of the API, not end-users)